api/src/main/java/com/epam/pipeline/manager/cloud/azure/AzureInstancePriceService.java

```diff
@@ -87,7 +88,7 @@ public double getSpotPrice(final String instanceType, final AzureRegion region)
     public double getPriceForDisk(final List<InstanceOffer> diskOffers, final int instanceDisk,
                                  final String instanceType, final AzureRegion region) {
         final InstanceOfferRequestVO requestVO = new InstanceOfferRequestVO();
-        requestVO.setTermType(ON_DEMAND_TERM_TYPE);
+        requestVO.setTermType(PriceType.ON_DEMAND.getName());
```

Comment:
Does Azure have the same prices for disks attached to  `on_demand` and `low_priority` VMs?

Reply:
Can't find any information about difference in this case. Assume that, yes.